salt/modules/nxos.py

```diff
@@ -1,49 +1,217 @@
 # -*- coding: utf-8 -*-
 """
-Execution module for Cisco NX OS Switches Proxy minions
+Execution module for Cisco NX OS Switches.
 
 .. versionadded:: 2016.11.0
 
-For documentation on setting up the nxos proxy minion look in the documentation
-for :mod:`salt.proxy.nxos <salt.proxy.nxos>`.
+This module supports execution using a Proxy Minion or Native Minion:
```

Comment:
We have an issue with docs here. 

I did look in the official sphinx docs for examples at a file level and didn't find anything that seemed applicable here.

Reply:
What's the docs issue?

Reply:
From the build: `/var/jenkins/workspace/pr-docs_PR-54931/salt/modules/nxos.py:docstring of salt.modules.nxos:7:Unexpected indentation.`

Reply:
```/var/jenkins/workspace/pr-docs_PR-54931/salt/modules/nxos.py:docstring of salt.modules.nxos:7:Unexpected indentation.```
it's line 7 of the docstring I'm pretty sure, not the file, so who knows which doc string it's from.

Reply:
```
/code/salt/modules/nxos.py:docstring of salt.modules.nxos:9: WARNING: Unexpected indentation.
/code/salt/modules/nxos.py:docstring of salt.modules.nxos:56: WARNING: Definition list ends without a blank line; unexpected unindent.
/code/salt/proxy/nxos.py:docstring of salt.proxy.nxos:63: WARNING: Definition list ends without a blank line; unexpected unindent.
/code/doc/topics/installation/nxos.rst:154: WARNING: Title underline too short.
```

Is the warnings I get when running without `-W` so you will likely hit those too

Reply:
```
diff --git a/doc/topics/installation/nxos.rst b/doc/topics/installation/nxos.rst
index a753c051ef..91c7bfecf3 100644
--- a/doc/topics/installation/nxos.rst
+++ b/doc/topics/installation/nxos.rst
@@ -151,7 +151,7 @@ GuestShell Salt Minion Installation
 This section is only required when running the SaltStack Minion from the ``guestshell``.
 
 STEP 1a: Enable the Guestshell on low footprint N3ks
----------------------------------------------------
+----------------------------------------------------
 
 **NOTE:** Skip down to **STEP 1b** if the target system is not a low footprint N3k.
 
diff --git a/salt/modules/nxos.py b/salt/modules/nxos.py
index 293508fcdc..8770004f5e 100644
--- a/salt/modules/nxos.py
+++ b/salt/modules/nxos.py
@@ -5,9 +5,9 @@ Execution module for Cisco NX OS Switches.
 .. versionadded:: 2016.11.0
 
 This module supports execution using a Proxy Minion or Native Minion:
-1) Proxy Minion: Connect over SSH or NX-API HTTP(S).
+   1) Proxy Minion: Connect over SSH or NX-API HTTP(S).
    See :mod:`salt.proxy.nxos <salt.proxy.nxos>` for proxy minion setup details.
-2) Native Minion: Connect over NX-API Unix Domain Socket (UDS).
+   2) Native Minion: Connect over NX-API Unix Domain Socket (UDS).
    Install the minion inside the GuestShell running on the NX-OS device.
 
 :maturity:   new
@@ -54,7 +54,7 @@ cookie
 
 save_config:
     If True, 'copy running-config starting-config' is issues for every
-        configuration command.
+    configuration command.
     If False, Running config is not saved to startup config
     Default: True
 
diff --git a/salt/proxy/nxos.py b/salt/proxy/nxos.py
index fc62bce7e2..3bd921273d 100644
--- a/salt/proxy/nxos.py
+++ b/salt/proxy/nxos.py
@@ -61,7 +61,7 @@ password:
 
 save_config:
     If True, 'copy running-config starting-config' is issues for every
-        configuration command.
+    configuration command.
     If False, Running config is not saved to startup config
     Default: True
 
```
seems to fix it.

Reply:
Thanks, that did it!